code/modules/clothing/shoes/clownshoes.dm

```diff
@@ -0,0 +1,204 @@
+/obj/item/clothing/shoes/clown_shoes
+	desc = "The prankster's standard-issue clowning shoes. Damn, they're huge! Ctrl-click to toggle waddle dampeners."
+	name = "clown shoes"
+	icon_state = "clown"
+	inhand_icon_state = "clown_shoes"
+	slowdown = SHOES_SLOWDOWN+1
+	pocket_storage_component_path = /datum/component/storage/concrete/pockets/shoes/clown
+	var/enabled_waddle = TRUE
+	lace_time = 20 SECONDS // how the hell do these laces even work??
+
+/obj/item/clothing/shoes/clown_shoes/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/clownstep1.ogg'=1,'sound/effects/clownstep2.ogg'=1), 50)
+
```

Comment:
```suggestion
/obj/item/clothing/shoes/clown_shoes/Examine()
	. = ..()
	. += "<span class='notice'>Ctrl-click to toggle waddle dampeners.</span> 
```

Then fix all the descriptions to not manually include this, thanks!

Reply:
![MrConflictBringer](https://user-images.githubusercontent.com/5155917/89697034-9aefb800-d91a-11ea-813a-a1aa48e4312e.png)